readthedocs/settings/docker_compose.py

```diff
@@ -142,6 +144,7 @@ def show_debug_toolbar(request):
     S3_MEDIA_STORAGE_BUCKET = 'media'
     S3_BUILD_COMMANDS_STORAGE_BUCKET = 'builds'
     S3_BUILD_ENVIRONMENT_STORAGE_BUCKET = 'envs'
+    S3_BUILD_LANGUAGES_STORAGE_BUCKET = 'languages'
```

Comment:
For consistency, perhaps this as well?

```suggestion
    S3_BUILD_LANGUAGES_STORAGE_BUCKET = 'build-languages'
```

I could also see this being more of a generic "build cache" rather than something very specific like "languages"/"build-languages"/"runtimes" -- though we already cache builds model objects and can cache build environments :grimacing:

Reply:
What is the benefit of putting all of these in the same bucket?

Reply:
Mostly to avoid bucket creep, we have a number of buckets that each require setup in prod and dev and there's not much of a  benefit in isolating tarballs of language runtimes from other assets used during build time.

Reply:
> there's not much of a benefit in isolating tarballs of language runtimes from other assets used during build time.

My reason to separate them is that I see "cached environment" as something volatile that it's not required for the build to work, and "tarballs of languages runtime" as _required_. So, if something happens to "cached environment", I don't care too much --but for the languages, we need to take action.

This affects any maintenance task that we may need to do over the buckets in the future.

Again, this separation is not strictly required, I just see this as something good to have separated due to the different importance they have.

(context: we mentioned this in the previous comment as well,https://github.com/readthedocs/readthedocs.org/pull/8453#discussion_r705139179)

Reply:
Yeah, we don't need to reuse the build environment cache bucket (`envs`), but I was more suggesting using something less specific than `languages`.  Thinking ahead, I'm sure there is something else we'll want to cache for build processes, and `languages` will be too specific to store it.

Combined with the overlap in meaning with locales and RTD languages, it's might be worth finding another way to describe "things we download into the builders during the build process".

Reply:
We are talking about using `build.tools` in the config file, so the bucket could be named `build-tools` as well, I guess.